services/galley/src/Galley/Schema/V89_TruncateMLSGroupMemberClient.hs

```diff
@@ -28,6 +28,6 @@ import Text.RawString.QQ
 -- could've been. This migration just deletes old data. This could break some
 -- conversations/users in unknown ways. But those are most likely test users.
 migration :: Migration
-migration = Migration 88 "Truncate mls_group_member_client" $ do
+migration = Migration 89 "Truncate mls_group_member_client" $ do
```

Comment:
Does this mean that this migration will run again on some instances?

Reply:
Will try putting the conflicting migrations into 1 file, as discuessed.